app/src/actions/groupActions.js

```diff
@@ -22,13 +23,15 @@ export const createGroup =(name) => dispatch => {
 
 export const getGroups =() => dispatch => { 
   dispatch({type:GETTING_GROUPS});
-  axios.get(`${URL}api/groups`)
+  axios.get(`${URL}api/groups`,{ headers: { Authorization: localStorage.getItem("token") }})
```

Comment:
if we had more time it might be worth checking to see if there is anything defined on local storage before making a server request